src/main/java/uk/gov/hmcts/ccd/domain/service/common/AuthorisedCaseDataFilter.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.ccd.domain.service.common;
```

Comment:
I'm a bit concerned about the number of small security related classes we are introducing recently. Classes that just group operations. 
I'd be tempted to remove this class and move this code and it's dependencies into AuthorisedCaseDetailsSearchOperation, even thought I agree AuthorisedCaseDetailsSearchOperation would become more complex